lib/create_poll.dart

```diff
@@ -108,6 +126,9 @@ class _OptionsList extends State<OptionsList> {
       child: Column(
         children: <Widget>[
           TextFormField(
+            onChanged: (value) {
+              print('Value for field option as "$value"' + value);
```

Comment:
Aren't you printing value twice? $value would print the value right?

Reply:
fixed